src/openforms/forms/models/form.py

```diff
@@ -491,7 +491,7 @@ def copy(self):
             else ""
         )
         copy.slug = _("{slug}-copy").format(slug=self.slug)
-        copy.product = None
+        copy.product = self.product
```

Comment:
I tried tracking down why this was originally done this way, but the commit message is not helpful and the original author no longer works for us :shrug: